jorgeacetozi/notepad
# [frontend] Create basic frontend for adding notes

## Changelog

- Create simple frontend consuming the rest endpoint to add new notes

## Tests

- Add a new note informing the title and content. It should save it in the database and display the note in the grid.
- Add a new note without title and content. It should show an error message and not add it.
